doc/source/whatsnew/v0.19.0.rst

```diff
@@ -235,17 +246,17 @@ converting to ``Categorical`` after parsing.  See the io :ref:`docs here <io.cat
 
 .. ipython:: python
 
-   data = 'col1,col2,col3\na,b,1\na,b,2\nc,d,3'
+   data = "col1,col2,col3\na,b,1\na,b,2\nc,d,3"
```

Comment:
I believe this can be converted into multiline too? @dsaxton @MarcoGorelli

Reply:
Yes, I think that would be good

Reply:
Sure. There's also a similar line in doc/source/whatsnew/v0.19.0.rst .

Just a generic comment - it doesn't seem to make a difference in these examples, but in general you'll probably want to remove the first newline by starting `"""` with `\`:
```python
In [7]: data = """
   ...: a,b,c
   ...: 1,Yes,2
   ...: """

In [8]: data
Out[8]: '\na,b,c\n1,Yes,2\n'

In [9]: data = """\
   ...: a,b,c
   ...: 1,Yes,2
   ...: """

In [10]: data
Out[10]: 'a,b,c\n1,Yes,2\n'
```

See how the second example (starting with `"""\` matches the string `'a,b,c\n1,Yes,2\n'` while the first one has a leading newline)

Reply:
In these examples the extra newlines don't affect the result

```python
In [15]: data = """\
    ...: a,b,c
    ...: 1,Yes,2\
    ...: """

In [16]: pd.read_csv(io.StringIO(data))
Out[16]:
   a    b  c
0  1  Yes  2

In [17]: data = """
    ...: a,b,c
    ...: 1,Yes,2
    ...: """

In [18]: pd.read_csv(io.StringIO(data))
Out[18]:
   a    b  c
0  1  Yes  2
```

so no need to add these `\` here, it's just something to be aware of in case you change to multi-line format in other parts of the docs and there is does make a difference

Reply:
Oh, I see! I'll add `\` to the ones I've updated so far then.

Reply:
Oh! Okay! Thank you, I'll remember that! Shall I see if the ones I updated are affected by this or not? And make changes accordingly?

Reply:
Pretty sure all the ones you updated here are the same as this one `(pd.read_csv(io.stringio(...` so I think we're fine here

As for whether there are any tools to do this automatically - I don't think so 😢 

If you update the line in ` line in doc/source/whatsnew/v0.19.0.rst` then I think this is good to go!

Reply:
Sorry, my bad, this _is_ v0.19.0, you updated it already.

Cool, looks good to me!